js_modules/dagit/packages/core/src/app/AppTopNav.tsx

```diff
@@ -108,16 +109,16 @@ const NavButton = styled.button`
   background: transparent;
   display: none;
 
-  .bp3-icon svg {
-    transition: fill 100ms linear;
+  ${IconWrapper} {
+    transition: color 100ms linear;
```

Comment:
Not 100% sure but I wonder if we should always animate color on the icon component?

Reply:
Possibly, maybe we can revisit in a followup after trying these out for a bit.